packages/shared/routes/dashboard/governance/views/GovernanceEventDetails.svelte

```diff
@@ -96,6 +108,54 @@
         )
         currentVoteValue = participation?.answers[0] ?? null
     }
+
+    const handleTransferState = (state: TransferState): void => {
+        if (!state) return
+
+        const _onCancel = () => {
```

Comment:
I'm not a huge fan of embedding functions in other functions. It makes code more difficult to read. Ideally this should be extracted. Also calling it `onCancel` instead of `_onCancel`, allows you to write less code on l.136

Reply:
I just noticed that the same is happening in `Staking.svelte`. We should aim to be consistent, so either leave it as it is, or modify both. If you pick the second approach, it might make sense to refactor the `handleTransferState` into the relevant repo, so that you just can import it throughout the codebase.

Reply:
I think it is a very personal choice and I like [closures](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Closures), should we discuss about creating a coding convention for it?

Reply:
I'll ask Matt about it, I guess closures are appropriate sometimes, but I'm ot sure if it makes sense to make a convention around it.